hack/codegen.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+set -eux
+
+IS_CONTAINER=${IS_CONTAINER:-false}
+ARTIFACTS=${ARTIFACTS:-/tmp}
+CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-podman}"
+
+if [ "${IS_CONTAINER}" != "false" ]; then
+  export XDG_CACHE_HOME=/tmp/.cache
+  eval "$(go env)"
+  cd "${GOPATH}"/src/github.com/metal3-io/cluster-api-provider-metal3
+  INPUT_FILES="\
+  config/certmanager/*.yaml
+  config/crd/*.yaml
+  config/crd/bases/*.yaml
+  config/crd/patches/*.yaml
+  config/default/*.yaml
+  config/manager/*.yaml
+  config/rbac/*.yaml
+  config/webhook/*.yaml
+  config/*.yaml
```

Comment:
I don't see `/bmo`. I'm not sure if it also needs to be added or not?

Reply:
this is v1a3, there is no BMO, and no ipam